### ISSUE TITLE
Testing p33-rhel8 vms

### ISSUE THREAD

**Issue by Sandeek** [norment/tsd_issues#67]
Please report any issues in logging and running programs in rhel8 machines here.

**Comment by ofrei** [comment]
1. I still have login issues with   ``p33-rhel8-01-pool`` (as described in  RT #4263641) - basically, first time login works well, after machine is locked due to few minutes of inactivity  I can't unlock it again. The second machine ( ``p33-rhel8-02-pool`` ) works  well, I don't have this issue there.
2. (solved) previously machines only had Norwegian keyboard layout. User could manually add English layout. With  ``p33-rhel8-02-pool``  this is done automatically.
3. (by design) there is no ``/cluster`` access on rhel8 machines. This is by design. The recommendation is to ``ssh -Y p33-appn-norment01``, i.e. use ``-Y`` for x11 forwarding, to enable GUI such as ``rstudio`` or ``matlab``. To use ``rstudio`` one additional step is needed here, which is to run ``module load libGLU`` before starting rstudio on a remote host. This is documented in [the wiki](https://wiki.norment.uiocloud.no/dokuwiki/tsd_software).
3. (minor issue, I haven't submitted it) rhel8 machines doesn't support scaling factor for  4K screen.

**Comment by zillurbmb51** [comment]
Hi,
I can not login from any of the pools (attached). After clicking any pool, a dark screen appears and few moments later it disappears.
![vmware-tsd-pools](https://user-images.githubusercontent.com/12833907/107781603-5a994880-6d48-11eb-8eb1-f0f6bed3492c.png)

**Comment by danielroelfs** [comment]
Even though it says "Norwegian" in the keyboard locale settings, Norwegian key entries are not recognised (e.g. a dash on my Norwegian keyboard would appear as a plus-sign in rhel8). This happens for both p33-rhel8-01-pool and p33-rhel8-02-pool.

**Comment by Sandeek** [comment]
@zillurbmb51 does this problem persist for you?

**Comment by zillurbmb51** [comment]
@Sandeek Yes. I just checked now and same problem occurs.

**Comment by Sandeek** [comment]
Update: the rhel8-01-pool was initially created with the configuration of p33-tl01-l but now the memory is increased to 64 GB and the rhel8-02-pool memory has been increased to 8 GB (which can be further increased if more users would be using it).

**Comment by HildeTJ** [comment]
Hi, I am having some troubles when I want to login with VMware both on p33-rhel8-01-pool and p33-rhel8-02-pool. I only get the screen that I have attached, and I can't get to the actual login where I can put my username p33-hildetj. It seems frozen not matter what I press on my keyboard. I'm not sure what I am doing wrong and is there something I can do to fix this? 
![Screen_login](https://user-images.githubusercontent.com/59874373/112144645-c8ebe900-8bd9-11eb-91a6-22f252cfba6c.jpg)

**Comment by pierreberthet** [comment]
I can not seem to be able to connect this morning, worked fine yesterday.
When I launch the p33-rhel8-01-pool I get a black screen, vmware appears to be frozen as the top panel does not show anymore upon mouse pointing. After an irregular interval (few seconds - minutes) it closes itself and goes back to machine selection (same window that Zillur showed).
rhel8-02-pool opens and closes right away.

**Comment by zillurbmb51** [comment]
Having issue with login p33 using thinlinc from this morning (attached).
![p33-march-26-morning-thinlinc](https://user-images.githubusercontent.com/12833907/112605302-726bee00-8e17-11eb-9686-9eeb3cdb6cd0.png)

**Comment by ofrei** [comment]
@zillurbmb51 Please submit a TSD-drift ticket. Is it OK if I remove your comment, to keep this issue focused on rhel8 testing? You are very welcome to submit a new issue about thinlink and paste there your TSD-drift RT ticket number.  The point of github issues is to inform other users about issues - but let's have it structured. thinlink issues and rhel8 issues are clearly independent problems.

**Comment by Sandeek** [comment]
@zillurbmb51 please check if you are able to login to thinlinc - the processes that were running in the previous session were killed - and as Alex said we can follow up through email/rt-ticket thread.

**Comment by zillurbmb51** [comment]
@Sandeek , Yes, login from thinlinc is ok now.

**Comment by pierreberthet** [comment]
> I can not seem to be able to connect this morning, worked fine yesterday.
> When I launch the p33-rhel8-01-pool I get a black screen, vmware appears to be frozen as the top panel does not show anymore upon mouse pointing. After an irregular interval (few seconds - minutes) it closes itself and goes back to machine selection (same window that Zillur showed).
> rhel8-02-pool opens and closes right away.

So it appears that the compositor manager selected by Gnome on my Linux system is involved. Using Wayland instead of X, VMWare can connect and all seem to work nicely. Selecting Gnome classic (so using X) leads to the problem I described. 
So most likely a VMWare issue? or is there an incompatibility with the remote machine display manager?

**Comment by fuyu00** [comment]
There are scaling issues with rhel-8 machines - the available resolutions don't play well with 3:2 aspect ratio monitors (3000x2000px) and the scaling option doesn't seem to change anything, rendering all UI elements incredibly small.

**Comment by ofrei** [comment]
@fuyu00 Hi Adriano
Do you have a 4K screen? What operating system do you have on your host machine? 
I'm having the same issue - currently discussing with TSD on an RT ticket. My workaround for now was to change the resolution on  my host machine.

**Comment by fuyu00** [comment]
Hi,
It's a 3000x2000 monitor on my laptop - unfortunately none of the resolutions available are ideal as they leave big black boxes empty. And if I set it to auto everything is quite tiny

**Comment by fuyu00** [comment]
It's a Windows 10  laptop - Surface Book

**Comment by ofrei** [comment]
Then it might be different from my issue. Do you already have an RT ticket / please submit one, and paste RT# here - I'll reference it on my ticket. 
Also, could you try the latest Horizon VMWare client? I've updated mine just today, now run VMWare version is 2103, build 8.2.0 (17759012); before that I had a very old version. This improved the performance quite a bit (I really noticed that), but unfortunately didn't fix the resolution. Either way if you run latest version it's easier to argue that there is an issue. I keep you in the loop about my conversation with TSD about my 4K issue.

**Comment by ofrei** [comment]
@fuyu00 For my ticket the solution was to set scaling in VMWare horizon as shown on screenshots below. The trick is to switch the "display" from "all monitors" to something like "full screen" - this enables the scaling option. Does this by chance solve your problem also? If not, as I mentioned before, please submit an RT ticket and copy the number here. 

step1
![image](https://user-images.githubusercontent.com/13171449/120993396-c5e9a900-c783-11eb-8ae2-c222a8d0da05.png)

step2
![image](https://user-images.githubusercontent.com/13171449/120993207-9aff5500-c783-11eb-9909-9323a2c1b78c.png)

**Comment by fuyu00** [comment]
Hi,
This is my current issue ticket [https://rt.uio.no/Ticket/Display.html?id=4408867](https://rt.uio.no/Ticket/Display.html?id=4408867)

I triesd the use all monitors options and it did not work, the machine tried to connect and then closed connection after a couple of seconds.

**Comment by ofrei** [comment]
@fuyu00
>I tried the use all monitors options and it did not work

My suggestion was to switch away from ``all monitors`` option (which is the default), and select e.g. ``full screen`` option. In this mode there will be a ``scaling`` setting which helped to solve my problem. Do you see ``scaling`` option?

**Comment by fuyu00** [comment]
Yes, that's what I have used previously. The scaling option doesn't seem to change anything for me, unfortunately.
